src/lib/mina/local-blockchain.ts

```diff
@@ -39,7 +40,23 @@ import {
   verifyAccountUpdate,
 } from './transaction-validation.js';
 
-export { LocalBlockchain };
+export { LocalBlockchain, TestAccount };
+
+class TestAccount extends PublicKey {
+  static random<N extends number = 1>(
+    count: N = 1 as never
+  ): N extends 1 ? TestAccount : TupleN<TestAccount, N> {
+    if (count === 1) return new this(PrivateKey.random()) as never;
+    return Array.from(
+      { length: count as number },
+      () => new this(PrivateKey.random())
+    ) as never;
+  }
+
+  constructor(readonly key: PrivateKey) {
+    super(key.toPublicKey());
```

Comment:
this is broken, the constructor has the two properties as arguments, you can use
```suggestion
    super.from(key.toPublicKey());
```

Reply:
This change is giving me the following error:

```
'super' must be called before accessing a property of 'super' in the constructor of a derived class.
```

Reply:
oh shoot, then just
```ts
let pk = key.toPublicKey();
super(pk.x, pk.isOdd);
```

Reply:
forgot that you must always call `super()`

Reply:
Is this okay?

Reply:
yep lgtm. the namespace doesn't conflict type-wise with the type of the same name?

Reply:
Correct

Reply:
and TS just merges the runtime function `TestPublicKey` and object `TestPublicKey` into the same value? I'm amazed

fwiw it also works to assign the function to a property (it modifies the type),
```ts
TestPublicKey.random = ...
```